components/gitpod-db/BUILD.yaml

```diff
@@ -96,6 +96,14 @@ packages:
       image:
         - ${imageRepoBase}/db-migrations:${version}
         - ${imageRepoBase}/db-migrations:commit-${__git_commit}
+  - name: latest-migration
+    type: generic
+    deps:
+      - :lib
+    config:
+      commands:
+        - ["sh", "-c", "node components-gitpod-db--lib/package/lib/typeorm/print-migration.js > latest-migration.json"]
+        - ["sh", "-c", "rm -r components*"]
```

Comment:
If we want to check in the `latest-migration.json`, could we make this a "code generation" task that we run as apret of `yarn build`. We would just run it every time we run the build.